app/components/TimeAgo.tsx

```diff
@@ -23,7 +22,7 @@ export const TimeAgo = ({
   const content = (
     <div className="flex flex-col">
       <span className="text-tertiary">{tooltipText}</span>
-      <span>{format(datetime, 'MMM d, yyyy p')}</span>
+      <span>{toLocaleDateString(datetime)}</span>
```

Comment:
This should include the time too. You might need another helper, or maybe `toLocaleString()` by itself would do it?

### `main`

![Screenshot 2024-04-16 at 7 08 51 PM](https://github.com/oxidecomputer/console/assets/3612203/dd2d8997-12f0-4415-8389-15a902220727)

### This branch

![Screenshot 2024-04-16 at 7 08 46 PM](https://github.com/oxidecomputer/console/assets/3612203/af89ba05-6525-43b9-8c98-05598290d5cf)

Reply:
The native formatting adds a comma after the year (for `en-US`), but I think it's fine
<img width="209" alt="Screenshot 2024-04-16 at 8 40 20 PM" src="https://github.com/oxidecomputer/console/assets/22547/5bbd8d72-6a4d-4073-9913-0fa244a541b0">